server/backend/api.go

```diff
@@ -837,6 +824,13 @@ func (b *Backend) AddMessageCreatedUpdate(userID, messageID string) error {
 	})
 }
 
+func (b *Backend) SetSystemLabelFilter(filter func(labelID string) bool) {
```

Comment:
```suggestion
func (b *Backend) SetSystemLabelFilter(filter SystemLabelFilter) {
```

Reply:
Good catch. I introduced the type in a later stage and forgot to change it here.